nmi_mysql/nmi_mysql.py

```diff
@@ -134,6 +149,8 @@ def _execute(self, _query, _params, _multi=False):
                         'affected_rows': result.rowcount
                     }
 
+            con.close()
```

Comment:
I think the connection is not really closed but it is just returned to the pool for other clients to use.
Because what if many clients get a connection from the pool and they haven't closed (returned) yet the connection to the pool, no other clients can get a connection right? Is my understanding correct?